### DISCUSSION TITLE
Scale for plant pots

### DISCUSSION THREAD

**Discussion by muelly2** [arendst/Tasmota#18903] [Show and tell]
Since I already killed several plants by not watering them in time, I felt there must be some plant alerting device in my home automation system.
First, I tried different humidity sensors based on capacitive or resistive measurement methods, but all these sensors failed afeter a while or didn't work at all.
In principle, the water contained in the plant pot can also be measured by weight, as the water consumed by the plant is mostly evaporated and the gain in weight of the plant due to growing is negelctable on a short time scale.
The device consists of a wooden plate with is rests on one side to the floor and is fitted with a load cell on the other side. The load cell is connected to its MX711 board, which in turn is connected to a D1 board. There is also a LED board with a WS2812 to indicate the status of the plant (ie. red, green, yellow).
![grafik](https://github.com/arendst/Tasmota/assets/61980635/1abddc9e-3d6d-454c-bd65-36446009c1f8)

The values obtained are send to my MQTT server and evaluated by the node red home automation system. In node red, the raw weight of the plant is rescaled to a value between 0-100 representing the fill level in percent. If the fill level goes below 5% the red color of the LED board gets engaged. Also a graph of all the plants is displayed on the dash board of node red.
![grafik](https://github.com/arendst/Tasmota/assets/61980635/0e7e0286-03a3-4431-893c-5f1645af7fee)

In contrast to the many other solutions, this one is very cost efficient as it only costs about EUR 10 (EUR 3 for the D1, EUR 7 for the load cell including board).

Best regards,
Mülly

**Comment by sfromis** [comment]
Of course, over time the growing plant is going to add weight, absorbing carbon from the air CO2, and water to form "building materials".

**Reply by muelly2** [reply]
You are completely right, the plants will get more weight due to growing. Therefore, one has additionally to consider the curvature of the weight over the time: If there is not enough moisture content in the plant pot, the decrease in weight will stagnate to a threshold (the net weight w/o water), comparable to a RC-circuit that gets discharged. From time to time you have to adjust the scaling (or implement some sort of mathematical model :-).
![Blumen_20230728](https://github.com/arendst/Tasmota/assets/61980635/2c164888-6dea-42d2-96e5-31b26e946247)
(green line depicts planr needing re-adjustement of scaling)
